eea/rdfmarshaller/browser/templates/license.pt

```diff
@@ -0,0 +1,5 @@
+<tal:license tal:define="license_text python:view.license_text">
```

Comment:
This file is not needed. You can render the script from the viewlet render() method